test/sequential/test-fs-readfile.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const common = require('../common');
+
+const assert = require('assert');
+const fs = require('fs');
+const path = require('path');
+
+// In case openssl is not available
+function slowRandomBytes(len) {
+  const buf = Buffer.alloc(len);
+  for (let i = 0; i < len; i++)
+    buf[i] = Math.random();
+  return buf;
+}
+
+common.refreshTmpDir();
+
+const fileInfo = [ { 'name': path.join(common.tmpDir, '1Kfile.txt'),
+                     'len': 1024,
+},
```

Comment:
Misaligned brace and you don't need to quote the property names, this isn't python. :-)

Reply:
> Misaligned brace

Linter fails if I indent the brace.